webpack-contrib/grunt-webpack
# Correct options

| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| Breaking change?  | no
| New feature?      | yes
| Deprecations?     | no
| Tests added?      | no
| Docs updated?     | yes 
| Fixed tickets     | Fixes #118, Fixes #71, Fixes #60
| License           | MIT

* Enable inline by default similar to webpack-dev-server cli
* Enable cache if watch enabled
* Remove check for array in dev-server-options, as it can never be an array
* Require webpack-dev-server 2.4.0 minimum
* Use logic from webpack-dev-server to create hot entries
